### PR TITLE
Make portfolio

### PR DESCRIPTION
**Pull request by LadyPython**

### CODE REVIEW

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -2,17 +2,21 @@
...
+    </header>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+      <h1 id="ladypython">LadyPython</h1>
+      <h2 id="feature">Love code & beauty</h2>
+      <p>I'm programmer with no experience</p>
```
Comment (pmueller-gg) [inline portfolio/src/main/webapp/index.html:17]: > with no experience
It's generally better to list some experience, e.g. from coursework :)

File: portfolio/src/main/webapp/images/logo.svg [hunk 2]
```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd">
+<svg width="100%" height="100%" viewBox="0 0 2732 2048" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" xml:space="preserve" xmlns:serif="http://www.serif.com/" style="fill-rule:evenodd;clip-rule:evenodd;stroke-linejoin:round;stroke-miterlimit:2;">
```
Comment (pmueller-gg) [inline portfolio/src/main/webapp/images/logo.svg:3]: https://google.github.io/styleguide/htmlcssguide.html#HTML_Line-Wrapping

> While there is no column limit recommendation for HTML, you may consider wrapping long lines if it significantly improves readability.

Comment (LadyPython) [reply portfolio/src/main/webapp/images/logo.svg:3]: But it's auto generated, I think. I uploaded this SVG file as image. Should I change it?

Comment (pmueller-gg) [reply portfolio/src/main/webapp/images/logo.svg:3]: If it's autogenerated it's ok. Optional: Mark it as a binary file in git. (e.g. https://www.programming-books.io/essential/git/identify-binary-files-cf8308a172e34112bfd725d57ab95f4d)

File: portfolio/src/main/webapp/contacts.html [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+        <li id="phone">+79259181314</li>
+        <li id="mail">sofia.ladypython@gmail.com</li>
+        <li id="telegram">@LadyPython</li>
+      </ul>
+      <!-- <p>or write me right now:</p>
+      <form id="contact" action="mailto:sofia.ladypython@gmail.com" method="post">
+        <textarea type="text" placeholder="Hire me..." onfocus="this.placeholder = ''" onblur="this.placeholder = 'Hire me...'"></textarea>
+        <button type="submit" form="contact" value="Send" onclick="sendForm('contact-form-container')">Send</button>
+      </form>
+      <div id="contact-form-container"></div> -->
```
Comment (pmueller-gg) [inline portfolio/src/main/webapp/contacts.html:25]: Please either fill out or remove for this commit (i.e., do not leave commented parts in the code).

**Pull request merged**